<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdaoMainWindow</class>
 <widget class="QMainWindow" name="AdaoMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>938</width>
    <height>478</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="ForumList">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>141</width>
      <height>461</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="Threadlist">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>0</y>
      <width>751</width>
      <height>461</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="nextpageButton">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>400</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>下一页</string>
    </property>
   </widget>
   <widget class="QPushButton" name="prepageButton">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>0</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>上一页</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>938</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
